chapter02/dtj/ex5

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/perl
+
+use warnings;
+
+print "Enter the string: ";
+$string = <STDIN>;
+print "Enter the number of repeats: ";
+chomp($num = <STDIN>);
+print $string x $num;
```

Comment:
- As pointed for exercise-4, some validation would be good.
- This will print the concatenated string in the same line. I thought we had to "prints out the string the number of times indicated by the number on separate lines". I might be mistaken.